frontend/src/static/js/main.js

```diff
@@ -40,6 +46,12 @@ window.app = new window.Vue({
       this.tabInfo.tabAuthorship = Object.assign({}, this.$store.state.tabAuthorshipInfo);
       this.activateTab('authorship');
     },
+    '$store.state.loadingOverlayCount': function () {
+      this.isLoadingOverlayEnabled = this.$store.state.loadingOverlayCount > 0;
+    },
+    '$store.state.loadingOverlayMessage': function () {
+      this.loadingOverlayMessage = this.$store.state.loadingOverlayMessage;
```

Comment:
This one can be simplified with a computed value so we do not need to keep the local copy.  See how other components use `mapState` for vuex values.